exercises/2/webapp/src/stories/NewsForm.stories.js

```diff
@@ -0,0 +1,17 @@
+import NewsForm from "../components/NewsForm.vue";
+import { action } from "@storybook/addon-actions";
+
+export default {
+  title: "NewsForm",
+  component: NewsForm,
+};
+
+const Template = (args, { argTypes }) => ({
```

Comment:
:star:   for creating a story for all of your components, locating them correcctly and   adding the @storybook/addon-essentials along with @storybook/addon-a11y